### PR TITLE
Eduard

### PR DESCRIPTION
**Pull request by Uprising5034**

### CODE REVIEW

File: planning/planning-so-i-do-not-fail.jpeg.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Plan
```
Comment (Satokii) [inline planning/planning-so-i-do-not-fail.jpeg.md:1]: Very detailed yet succinct plan, definitely one that will not cause you to fail!

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import { Route, Routes } from "react-router";
+
+import Header from "./components/Header/Header";
+import Home from "./components/Home/Home";
```
Comment (Satokii) [inline src/App.jsx:5]: I can already tell from the get go you're not messing about with every 'component should  be it's own component'

File: src/components/Home/components/PostFeed/PostItem/components/PostCommentFeed/PostCommentFeed.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
```
Comment (Satokii) [inline src/components/Home/components/PostFeed/PostItem/components/PostCommentFeed/PostCommentFeed.jsx:1]: Reading through the code of each of the individual components, it's all very clear and easy to follow along, I don't quite know what to suggest as it's all great as far as I can see!

File: src/components/Loader/PulseLoader.jsx [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export default function PulseLoader() {
```
Comment (Satokii) [inline src/components/Loader/PulseLoader.jsx:1]: Ah I kept seeing you import and use this component throughout but wasn't sure what it was. Makes a lot more sense now 😂

**Review by Satokii — COMMENTED** [review]
Very clean code throughout, great to read!

File: planning/planning-so-i-do-not-fail.jpeg.md (hunk 1 above)
Comment (Uprising5034) [reply planning/planning-so-i-do-not-fail.jpeg.md:1]: Thanks, have to admit having a checkbox to tick off was nice to have. Maybe I should take some effort planning these things...

File: src/App.jsx (hunk 2 above)
Comment (Uprising5034) [reply src/App.jsx:5]: Yeah I've not considered it till now but it is quite a few imports I've got there. 🤣

File: src/components/Home/components/PostFeed/PostItem/components/PostCommentFeed/PostCommentFeed.jsx (hunk 3 above)
Comment (Uprising5034) [reply src/components/Home/components/PostFeed/PostItem/components/PostCommentFeed/PostCommentFeed.jsx:1]: Thanks!

File: src/components/Loader/PulseLoader.jsx (hunk 4 above)
Comment (Uprising5034) [reply src/components/Loader/PulseLoader.jsx:1]: One of my stretch goals is to give all the elements a fancy loading widget once (and if) I finish everything else up. That component there is just to remind me of that.

File: src/components/Home/components/PostFeed/PostItem/components/PostCommentFeed/PostCommentFeed.jsx (hunk 3 above)
Comment (Satokii) [reply src/components/Home/components/PostFeed/PostItem/components/PostCommentFeed/PostCommentFeed.jsx:1]: Oh for sure, could literally spend all day reading through the code to fully understand it all, let alone 15 minutes lol

File: src/components/Loader/PulseLoader.jsx (hunk 4 above)
Comment (Satokii) [reply src/components/Loader/PulseLoader.jsx:1]: Haha with the name you'd given that I figured you would have some sort of fancy effect going on during the load time and such but couldn't see where you'd coded it. Yeah makes sense, will be quite cool if you get around to implementing that!

**Pull request closed without merging**